azure/flow-runner-function/README.md

```diff
@@ -34,7 +34,7 @@ Then deploy your function (this can take many seconds to finish):
 
 Then get your HTTP trigger URL and test the endpoint - your URL should be similar to what's shown below:
 
-    curl -w "\n" https://az-functions-20200227122602814.azurewebsites.net/api/HttpExampleDhfFlow --data flowNameDoesntMatterYet
+    curl -w "\n" https://az-functions-dhf-sample-v1.azurewebsites.net/api/HttpExampleDhfFlow --data flowNameDoesntMatterYet
```

Comment:
Small nit - can you name is "data-hub-sample" instead of "dhf-sample"? We try to use "data-hub" instead of "dhf" wherever possible.

Reply:
Sure. Done